docs/user-guide/mobile-sdk.md

```diff
@@ -0,0 +1,37 @@
+All SDK libraries available as pre-built packages on the GitHub [release page](https://github.com/mysteriumnetwork/node/releases).
+It can be downloaded and installed manually with any preferred way.
+
+But we also support several standard ways for installing SDK using dependency managers.
+
+## iOS
+
+It is possible to get iOS SDK library using [Carthage](https://github.com/Carthage/Carthage) dependency manager.
+
+You can follow the [Carthage quick start guide] to get familiar how to use it or just use the following steps:
+
+1. Carthage should be installed in the system.
+2. `Cartfile` should be created in the same directory where `.xcodeproj` or `.xcworkspac` located.
```

Comment:
2. `Cartfile` should be created in the same directory where `.xcodeproj` or `.xcworkspac` is located.